crates/bindings-csharp/Codegen.Tests/fixtures/diag/Lib.cs

```diff
@@ -0,0 +1,328 @@
+﻿using SpacetimeDB;
+
+public enum LocalEnum { }
+
+[SpacetimeDB.Type]
+public partial struct TestUnsupportedType
+{
+    public DateTime UnsupportedSpecialTime;
```

Comment:
Shouldn't this say UnsupportedSpecialType?

Reply:
Well this is a `DateTime` property, so I figured "Time" naming would be slightly clearer. But I can see how this can look like a typo :)

Reply:
Yeah I think it's best since it matches the SpecialType enum we're seeing in codegen